Xamarin.PropertyEditing.Mac/Controls/Custom/FocusableComboBox.cs

```diff
@@ -5,6 +5,25 @@ namespace Xamarin.PropertyEditing.Mac
 {
 	internal class FocusableComboBox : NSComboBox
 	{
+		public RowProxyResponder ProxyResponder { get; set; }
+
+		public override void KeyDown (NSEvent theEvent)
+		{
+			switch (theEvent.KeyCode) {
+			case (int)NSKey.Tab:
```

Comment:
for the tab keys it's better to handle the responders "insertTab:" and "insertBacktab:" instead of interfering with the keyDown event.

Reply:
insertTab: only applies to NSTextView based fields, no? 
https://developer.apple.com/documentation/appkit/nsstandardkeybindingresponding?language=objc